site/source/partials/header.hbs

```diff
@@ -27,14 +27,41 @@
   <!-- stylesheet -->
   <link rel="stylesheet" href="{{assets}}css/style.css">
 
-  <!-- Cedar -->
-
-  <!-- esri Cedar -->
-  <script type="text/javascript" src="http://square.github.io/crossfilter/d3.v3.min.js"></script>
-  <script type="text/javascript" src="{{assets}}js/vega.js"></script>
+  <!-- Cedar and dependencies (d3, vega) -->
+  <!-- NOTE: the bar chart examle does not work w/ latest vega (v1.5+), so here we are pulling in an oldder version (v1.4.3) -->
```

Comment:
Damnit, meant to say pie chart example doesn't work w/ latest vega (not bar chart). Prob ought to look into that.